<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleship</class>
 <widget class="QDialog" name="battleship">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>331</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="GameGrid">
    <item row="1" column="1">
     <widget class="QPushButton" name="Coll2R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Coll3R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QPushButton" name="Coll10R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="Coll7R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="Coll1R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="Coll9R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QPushButton" name="Coll10R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <widget class="QPushButton" name="Coll10R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QPushButton" name="Coll10R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="Coll10R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Coll1R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="Coll6R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QPushButton" name="Coll10R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="Coll1R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Coll3R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="Coll6R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Coll1R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Coll4R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="Coll9R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="Coll5R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Coll1R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Coll1R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Coll1R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="9">
     <widget class="QPushButton" name="Coll10R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="Coll8R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="Coll5R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Coll1R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="9">
     <widget class="QPushButton" name="Coll10R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="Coll10R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="Coll5R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="Coll1R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="Coll10R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Coll4R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Coll1R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="Coll7R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="Coll8R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Coll5R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="Coll6R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="Coll7R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="Coll8R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QPushButton" name="Coll9R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="Coll9R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="Coll9R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="Coll9R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="Coll8R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="Coll8R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="Coll7R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="Coll7R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="Coll6R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="Coll6R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="Coll5R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="Coll4R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Coll4R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Coll3R2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Coll3R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Coll2R1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Coll2R3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Coll2R4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Coll2R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Coll3R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Coll4R5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Coll2R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Coll3R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Coll4R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Coll5R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="Coll6R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="Coll7R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="Coll8R6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Coll2R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Coll3R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Coll2R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Coll3R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="Coll2R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Coll4R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Coll5R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="Coll6R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="Coll7R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="Coll8R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QPushButton" name="Coll9R7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="8">
     <widget class="QPushButton" name="Coll9R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="8">
     <widget class="QPushButton" name="Coll9R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="8">
     <widget class="QPushButton" name="Coll9R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QPushButton" name="Coll8R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="Coll8R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="Coll8R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="Coll7R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="Coll7R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QPushButton" name="Coll7R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="Coll6R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="Coll6R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="Coll6R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="Coll5R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="Coll5R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="Coll5R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="Coll4R8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="Coll4R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="Coll4R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="Coll3R9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="Coll2R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="Coll3R10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
